pandas/tests/extension/arrow/test_bool.py

```diff
@@ -53,6 +53,16 @@ def test_from_sequence_from_cls(self, data):
 
 
 class TestReduce(base.BaseNoReduceTests):
+    @pytest.mark.parametrize("skipna", [True, False])
+    def test_reduce_series_numeric(self, data, all_numeric_reductions, skipna):
+        op_name = all_numeric_reductions
+        if op_name in ("max", "min"):
+            pass
+        else:
+            ser = pd.Series(data)
+            with pytest.raises(TypeError):
```

Comment:
Error message?

Reply:
Thanks for your review.
I think this is non-sense to implement `argmax`, `argmin`, `max` and `min` for `ArrowBoolArray` which have two values only. Calling those methods will raise `NotImplementedError`. Getting but not calling the `max` and `min` attributes by `gettattr` doesn't raise any error.

Reply:
I think @gfyoung was asking that you assert something about the error message with the `match=` keyword.

Reply:
Those function just raise the `TypeError` without any error message.